proto/viam/app/v1/app.proto

```diff
@@ -378,6 +378,15 @@ message DeleteOrganizationMemberResponse {}
 // Location
 //
 
+// Used for rendering an organization's information on the frontend (limited
+// to id, name, or both).
+message OrganizationIdentity {
+  // Organization ID.
+  string organization_id = 1;
+  // Organization name.
+  string organization_name = 2;
```

Comment:
making these `organization_id` and `organization_name` are somewhat redundant. could could probably make it `id` and `name` instead